parm/gdas/snow_stage_ens_update.yaml.j2

```
@@ -0,0 +1,113 @@
+######################################
+# set some variables
+######################################
+{% set gPDY = previous_cycle | to_YMD %}
+{% set gcyc = previous_cycle | strftime("%H") %}
+{% set PDY = current_cycle | to_YMD %}
+{% set cyc = current_cycle | strftime("%H") %}
+{% if DOIAU %}
+{% set bkg_time = SNOW_WINDOW_BEGIN | to_fv3time %}
+{% else %}
+{% set bkg_time = current_cycle | to_fv3time %}
+{% endif %}
```

Suggest tabbing for readability:
```suggestion
{% if DOIAU %}
    {% set bkg_time = SNOW_WINDOW_BEGIN | to_fv3time %}
{% else %}
    {% set bkg_time = current_cycle | to_fv3time %}
{% endif %}
```